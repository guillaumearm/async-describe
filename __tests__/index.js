const term = require('terminal-kit').terminal

describe('async-describe', () => {
  test('demo', (done) => {
    require('../demo');
    setTimeout(() => {
      expect(term.getActions()).toMatchSnapshot();
      term.resetActions();
      done();
    }, 1000)
  });
});
