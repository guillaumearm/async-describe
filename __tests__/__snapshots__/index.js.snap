// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`async-describe demo 1`] = `
Array [
  Object {
    "payload": Array [
      "tests
",
    ],
    "type": "BOLD",
  },
  Object {
    "payload": Array [
      "  build client
",
    ],
    "type": "BOLD",
  },
  Object {
    "payload": Array [
      "     test",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      " ✓",
    ],
    "type": "YELLOW",
  },
  Object {
    "payload": Array [
      "
",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      "  start server
",
    ],
    "type": "BOLD",
  },
  Object {
    "payload": Array [
      "     test 1",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      " ✓",
    ],
    "type": "YELLOW",
  },
  Object {
    "payload": Array [
      "
",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      "     test 2",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      " ✓",
    ],
    "type": "YELLOW",
  },
  Object {
    "payload": Array [
      "
",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      "  additional test I
",
    ],
    "type": "BOLD",
  },
  Object {
    "payload": Array [
      "  additional test II
",
    ],
    "type": "BOLD",
  },
  Object {
    "payload": Array [
      "     test3",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      " ✓",
    ],
    "type": "YELLOW",
  },
  Object {
    "payload": Array [
      "
",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      "  run functional tests
",
    ],
    "type": "BOLD",
  },
  Object {
    "payload": Array [
      "     test4",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      "[ERROR]",
    ],
    "type": "RED",
  },
  Object {
    "payload": Array [
      "
",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      "[ERROR]",
    ],
    "type": "RED",
  },
  Object {
    "payload": Array [
      "     test4",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      "[ERROR]",
    ],
    "type": "RED",
  },
  Object {
    "payload": Array [
      "
",
    ],
    "type": "NORMAL",
  },
  Object {
    "payload": Array [
      "[ERROR]",
    ],
    "type": "RED",
  },
  Object {
    "payload": Array [
      "  more test
",
    ],
    "type": "BOLD",
  },
  Object {
    "payload": Array [
      "[ERROR]",
    ],
    "type": "RED",
  },
  Object {
    "payload": Array [
      "  more test
",
    ],
    "type": "BOLD",
  },
  Object {
    "payload": Array [
      "  more test
",
    ],
    "type": "BOLD",
  },
]
`;
